ui/app/templates/components/attributes-table.hbs

```diff
@@ -1,11 +1,16 @@
-<table class="table is-striped is-fixed is-compact is-darkened" ...attributes>
+<table class="table is-striped is-fixed is-compact is-darkened no-mobile-condense" ...attributes>
   <thead>
     <tr>
       <th class="is-one-third">Name</th>
       <th>Value</th>
     </tr>
   </thead>
   <tbody>
-    <AttributesSection @attributes={{@attributePairs}} />
+    <AttributesSection @attributes={{@attributePairs}}
+    @editable={{@editable}}
+    @onKVEdit={{@onKVEdit}}
+    @onKVSave={{@onKVSave}}
```

Comment:
`nit`: we're prop drilling these down 4 components. We could use a [closure component](https://guides.emberjs.com/release/components/block-content/) in `AttributesSection` to yield these values out? Adding a new function that floats down to `MetadataEditor` would make this a little challenging to track down.

Reply:
I ended up playing with this idea a bit yesterday and no matter how I worked it, it felt either a little less legible or a bit clunky. I'm going to opt to leave these as passed functions. The names are specific enough that they will be easy to track down.